src/main/java/sg/ncl/testbed_interface/TeamUsageInfo.java

```diff
@@ -0,0 +1,18 @@
+package sg.ncl.testbed_interface;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Created by dcszwang on 10/31/2016.
+ */
+@Getter
+@Setter
+public class TeamUsageInfo {
+
+    public String id;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make id a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)